src/braket/pennylane_plugin/ops.py

```diff
@@ -87,7 +87,7 @@ class CPhaseShift(Operation):
 
         \frac{d}{d \phi} \mathtt{CPhaseShift}(\phi)
         = \frac{1}{2} \left[ \mathtt{CPhaseShift}(\phi + \pi / 2)
-            + \mathtt{CPhaseShift}(\phi - \pi / 2) \right]
+            - \mathtt{CPhaseShift}(\phi - \pi / 2) \right]

```

Comment:
There seems to be a sign mismatch here for each operation for which the two-term parameter shift rule is defined. See [Mitarai et al. (2018)](https://arxiv.org/pdf/1803.00745.pdf) eq. 3. and the paragraph below.

Reply:
Note that we now have [`qml.ControlledPhaseShift`](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.ControlledPhaseShift.html) available in PL core. Is this the same definition? If so, we could deprecate it here in favour of the core op.

Reply:
Actually we now support XX, YY, and ZZ below as well.

Reply:
Good point! Except for YY, these will be included in the release (that one will extend into the next). [ISWAP](https://github.com/PennyLaneAI/pennylane/blob/ca227436391adf10df1f946cc727a9abb36673eb/pennylane/ops/qubit.py#L545) is available too.